web/src/message_helper.js

```diff
@@ -25,6 +25,7 @@ export function process_new_message(message) {
 
     message_store.set_message_booleans(message);
     message.sent_by_me = people.is_current_user(message.sender_email);
+    message.message_reactions = message.reactions || [];
```

Comment:
This seems very suspect to me; can you talk about what made you think this change should be made?

`reactions.js` has some logic involving `clean_reactions` that may be related.

Reply:
Yeah, that seems OK, with a comment, as we'd just be doing that normal processing a little early.